Wrapper/Throttle.php

```diff
@@ -79,6 +79,10 @@ private function throttle($name, $args, $counter = 1)
             $this->wait(1);
         } catch (\Doofinder\Api\Management\Errors\IndexingInProgress $e) {
             $this->wait(3);
+        } catch (\Doofinder\Api\Management\Errors\NotFound $e) {
```

Comment:
I think this should take into account if the method called is the deleteType one... if you try to index an item and the datatype does not exist the error should be thrown...